barter-data-rs/src/subscription/intent_order.rs

```diff
@@ -19,13 +20,10 @@ pub enum IntentOrderUpdate {
 #[derive(Clone, PartialEq, PartialOrd, Debug, Deserialize, Serialize)]
 pub struct IntentOrder {
     pub event: IntentOrderUpdate,
+    pub instrument: Instrument,
     pub id: String,
-    pub in_token: String,
-    pub in_token_addr: String,
-    pub in_amount: f64,
-    pub out_token: String,
-    pub out_token_addr: String,
-    pub out_amount: f64,
+    pub amount: f64,
+    pub buy: bool,
     pub start_ask: f64,
```

Comment:
re: discussion from yesterday - we still want the original amount in/out as u256 so we can execute the fill